google/cloud/forseti/scanner/scanners/base_scanner.py

```diff
@@ -115,3 +118,162 @@ def _output_results_to_db(self, violations):
             violation_access.session, inventory_index_id,
             index_state=IndexState.RUNNING)
         violation_access.create(violations, scanner_index_id)
+
+
+class KeBaseScanner(BaseScanner):
```

Comment:
Can you please move this to its own file?  `ke_base_scanner.py`?  This prevents us overloading a py file with too many classes.